backend/src/main/java/com/bakdata/conquery/models/events/stores/root/ColumnStore.java

```diff
@@ -12,15 +14,28 @@
  * Base-class representing the data of a single {@link com.bakdata.conquery.models.datasets.Column} of a {@link com.bakdata.conquery.models.events.Bucket}.
  * <p>
  * Internal representation of data is completely transparent to the caller. get may only be called if has is true.
- *
+ * <p>
  * This class has subclasses per storage-type and cannot get/set itself for type-safety reasons.
- *
+ * <p>
  * Every root class must have an associated {@link MajorTypeId} and {@link com.bakdata.conquery.models.preproc.parser.Parser}.
  */
 @JsonTypeInfo(use = JsonTypeInfo.Id.CUSTOM, property = "type")
 @CPSBase
 public interface ColumnStore {
 
+
+	/**
+	 * If necessary the given bucket will be set in store as parent
+	 *
+	 * @param bucket bucket that should be set as parent
+	 */
+	// BackReference set here because Jackson does not support for fields in interface see also https://github.com/FasterXML/jackson-databind/issues/3304
```

Comment:
```suggestion
	* @implNote BackReference set here because Jackson does not support for fields in interface see also https://github.com/FasterXML/jackson-databind/issues/3304
	*/
```

Reply:
Möchtest du nicht?

Reply:
angenehmer wäre ja direkt an einem Feld der Klasse `com.bakdata.conquery.models.events.stores.specific.DateRangeTypeCompound` wie wir uns zum ersten Mal ausgedacht haben.

Reply:
Ich habe nur den Kommentar aufgeräumt

Reply:
achso ich dachte, du kommentiertest über das BackReference